ui/css/perf.css

```diff
@@ -409,20 +409,8 @@ button:disabled {
 }
 
 .item-badge {
-  display: inline-block;
-  font-size: 12px;
-  font-weight: 600;
   padding: 3px 6px;
   color: #17a2b8;
```

Comment:
I think you can replace this`item-badge` class with the bootstrap color and padding utilities (`text-info`).